tutorials/asr/ASR_CTC_Language_Finetuning.ipynb

```diff
@@ -1,25 +1,12 @@
 {
-  "nbformat": 4,
-  "nbformat_minor": 0,
```

Comment:
Notebook changes need to be done in Colab not Jupiter otherwise it removes Colab features.

Reply:
tdil